src/pages/who-we-are/index.tsx

```diff
@@ -50,6 +53,15 @@ const AboutUs = ({ data }: TWhoWeAre) => {
         our_locations,
         banner,
     } = data?.strapiWhoWeArePage || {}
+
+    const clickToScrollHandler = useScrollToElement('investors-in-people')
+    const pathname_hash = getLocationHash()
+    useEffect(() => {
+        setTimeout(() => {
+            pathname_hash?.includes('investors-in-people') && clickToScrollHandler()
+        }, 1000)
+    }, [pathname_hash])
```

Comment:
we have no extra re-renders here so we don't need a useEffect 
better to use this approach. please check the behaviour
```suggestion
        const pathname_hash = getLocationHash();
    const clickToScrollHandler = useScrollToElement('investors-in-people')

    if (pathname_hash?.includes('investors-in-people')) {
        clickToScrollHandler();
    }
```